lib/components/src/Zoom/ZoomElement.tsx

```diff
@@ -15,6 +24,35 @@ const ZoomElementWrapper = styled.div<{ scale: number; height: number }>(({ scal
         transform: `scale(${1 / scale})`,
       }
 );
+
+const useMutationObserver = ({
+  element,
+  options = {},
+  callback,
+}: {
+  element: RefObject<Element>;
+  options: MutationObserverInit;
+  callback: MutationCallback;
+}): void => {
+  const observer = useMemo(
+    () =>
+      typeof MutationObserver === 'function'
```

Comment:
I don't think we need this check, `MutationObserver` has been available in all browsers since 2014. Save for Opera Mini which frankly nobody is going to be using to view their Storybook in anyway.

Reply:
I strongly **agree** with this, Storybook 7.0 will target `chrome100`. If it's supported in there, don't bother downgrading, transpiling, of feature detecting!